pkg/kubelet/container/testing/fake_runtime_helper.go

```diff
@@ -58,6 +59,14 @@ func (f *FakeRuntimeHelper) GeneratePodHostNameAndDomain(pod *v1.Pod) (string, s
 	return f.HostName, f.HostDomain, f.Err
 }
 
+func (f *FakeRuntimeHelper) GetHostNameForKernel(hostname string, hostDomainName string, hostnameFQDN bool) (string, error) {
```

Comment:
Should this be GetNodenameForKernel since it bridges pod's hostname to kernel's nodename?

Reply:
sounds good.